app/src/main/java/com/jalasoft/routesapp/ui/auth/login/viewModel/LoginViewModel.kt

```diff
@@ -102,55 +102,36 @@ constructor(private val repository: UserRepository) : ViewModel() {
         return isValid
     }
 
-    /*private fun validateEmail(email: String): Boolean {
-        var isValid = true
-        userManager.validateEmailUser(email, { users ->
-            if (users.isNotEmpty()) {
-                isValid = false
-            }
-        }, { error ->
-            Log.d(ContentValues.TAG, error)
-        })
-        return isValid
-    }*/
-
-    /*
-    fun permissionGranted(email: String, provider: LoginViewModel.ProviderType){
-        fragmentTransaction.replace(com.jalasoft.routesapp.R.id.action_loginFragment_to_homeFragment, LoginFragment())
-        putExtra()
-        fragmentTransaction.commit()
-    }*/
-
-    fun validateEmailGoogle(name: String, email: String, typeLogin: UserTypeLogin, credential: AuthCredential) = viewModelScope.launch {
+    fun validateEmailGoogleOrFacebook(name: String, email: String, typeLogin: UserTypeLogin, credential: AuthCredential) = viewModelScope.launch {

```

Comment:
this function is same as in RegisterUserViewModel right?, if yes make the logic similar

Reply:
check this function, i see that logic is the same as in RegisterUserViewModel but the way you implement is different in both maybe you can implement the same way in both